<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Igor Malinovskiy</author>
 <class>demoClass</class>
 <widget class="QMainWindow" name="demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Redis Desktop Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/demo.qrc">
    <normaloff>:/images/redisBigIcon.png</normaloff>:/images/redisBigIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: tahoma, arial, ubuntu, sans-serif;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="leKeySearchPattern">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;p&gt;&lt;b&gt;Key search supports regular expressions:&lt;/b&gt;&lt;/p&gt;

&lt;pre&gt;
. 	Any character except newline.
\. 	A period (and so on for \*, \(, \\, etc.)
^ 	The start of the string.
$ 	The end of the string.
\d,\w,\s 	A digit, word character [A-Za-z0-9_], or whitespace.
\D,\W,\S 	Anything except a digit, word character, or whitespace.
[abc] 	Character a, b, or c.
[a-z] 	a through z.
[^abc] 	Any character except a, b, or c.
aa|bb 	Either aa or bb.
? 	Zero or one of the preceding element.
* 	Zero or more of the preceding element.
+ 	One or more of the preceding element.
{n} 	Exactly n of the preceding element.
{n,} 	n or more of the preceding element.
{m,n} 	Between m and n of the preceding element.
??,*?,+?,
{n}?, etc. 	Same as above, but as few as possible.
(expr) 	Capture expr for use with \1, etc.
(?:expr) 	Non-capturing group.
(?=expr) 	Followed by expr.
(?!expr) 	Not followed by expr.
&lt;/pre&gt;
</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string>key search...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="pbFindFilter">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Apply filter</string>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/demo.qrc">
              <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="pbClearFilter">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Reset filter</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/demo.qrc">
              <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ConnectionTreeView" name="serversTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>8000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: monospaced;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>300</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QToolButton" name="pbImportConnections">
            <property name="minimumSize">
             <size>
              <width>145</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>145</width>
              <height>35</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>145</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Manage Connections</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/demo.qrc">
              <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="pbAddServer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>135</width>
              <height>35</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: 50px;

</string>
            </property>
            <property name="text">
             <string>Add New Connection</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/demo.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>580</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="rdmTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50000</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="icon">
         <iconset resource="../Resources/demo.qrc">
          <normaloff>:/images/redisIcon.png</normaloff>:/images/redisIcon.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Redis Desktop Manager</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="applicationIntroWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>570</width>
             <height>255</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="applicationInfoLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>190</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>190</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>190</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;

&lt;style&gt;a {color: darkblue; text-decoration: none}&lt;/style&gt;
&lt;body style=&quot;width:100%;&quot;&gt;&lt;div style=&quot;font-size:20pt; padding-top:10px;&quot;&gt;&lt;img style=&quot;float:left;&quot; src=&quot;:/images/logo.png&quot;/&gt;&lt;div style=&quot; font-size:24pt; margin-left: 100px; margin-top:2px; display: inline-block; padding-left:50px;&quot;&gt;Redis Desktop Manager&lt;/div&gt;&lt;div style=&quot;margin-left: 100px; display: inline-block; padding-left:53px; font-size: 11px; line-height: 21px&quot;&gt;&lt;b&gt;Cross-platform GUI application for managing Redis.&lt;/b&gt; &lt;br /&gt;&lt;b&gt;Version&lt;/b&gt; %VERSION% &amp;nbsp;&amp;nbsp;&amp;nbsp; Developed by - &lt;a href=&quot;http://glide.name/?rdm&quot;&gt;Igor Malinovskiy&lt;/a&gt; &lt;/div&gt;

&lt;div style=&quot;font-size: 10px&quot;&gt;Third Party Libs and Images:&lt;a href=&quot;http://qt-project.org/&quot;&gt; QT (GPL License)&lt;/a&gt;; &lt;a href=&quot;http://libqxt.org/&quot;&gt;Qxt (GPL License)&lt;/a&gt; , &lt;a href=&quot;http://www.libssh2.org&quot;&gt;libssh2 (BSD License)&lt;/a&gt;, &lt;a href=&quot;http://www.carlosprioglio.com/&quot;&gt;Redis Logo&lt;/a&gt;&lt;/div&gt;

&lt;div style=&quot;font-size: 10px;&quot;&gt;Thanks to:&amp;nbsp;&amp;nbsp;
&lt;a href=&quot;https://github.com/stalker-druid/&quot;&gt;Alexander&amp;nbsp;Arshinov&lt;/a&gt;, 
&lt;a href=&quot;https://github.com/loginwashere&quot;&gt;Dmitriy&amp;nbsp;Savchenko&lt;/a&gt;, 
&lt;a href=&quot;https://github.com/Dyachenko&quot;&gt;Yevgeniy&amp;nbsp;Dyachenko&lt;/a&gt;,
&lt;a href=&quot;https://github.com/AntonShevchuk&quot;&gt;Anton&amp;nbsp;Shevchuk&lt;/a&gt;, &lt;br /&gt;
&lt;a href=&quot;https://github.com/leonko&quot;&gt;Leonid&amp;nbsp;Batizhevsky&lt;/a&gt;,
&lt;a href=&quot;https://github.com/mrThe&quot;&gt;mrThe&lt;/a&gt;
&lt;/div&gt;


&lt;/div&gt;&lt;div style=&quot;clear: both;&quot;&gt;
&lt;hr style=&quot;border-top: 1px solid darkgrey; height: 5px; width: 100%;&quot; /&gt;
&lt;div  style=&quot; margin-left: 15px; vertical-align: bottom; font-size: 13px;&quot;&gt;
&lt;a style=&quot; color: red;&quot; href=&quot;https://github.com/uglide/RedisDesktopManager/issues&quot;&gt;&lt;img  src=&quot;:/images/alert.png&quot; /&gt;&amp;nbsp;Report issue&lt;/a&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;
&lt;a style=&quot;color: green; &quot; href=&quot;https://github.com/uglide/RedisDesktopManager/wiki&quot;&gt;&lt;img  src=&quot;:/images/help.png&quot; /&gt;&amp;nbsp;Documentation&lt;/a&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;a style=&quot;font-size: 13px; &quot; href=&quot;https://twitter.com/RedisDesktop&quot;&gt;&lt;img  src=&quot;:/images/twi.png&quot; /&gt;&amp;nbsp;Follow @RedisDesktop&lt;/a&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;a style=&quot;font-size: 13px; &quot; href=&quot;https://github.com/uglide/RedisDesktopManager/stargazers&quot;&gt;&lt;img  src=&quot;:/images/github.png&quot; /&gt;&amp;nbsp;Star!&lt;/a&gt;
&lt;/div&gt;
&lt;/div&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionTreeView</class>
   <extends>QTreeView</extends>
   <header>ConnectionTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/demo.qrc"/>
 </resources>
 <connections/>
</ui>
